src/main/java/de/zalando/aruha/nakadi/controller/CursorsController.java

```diff
@@ -37,22 +33,34 @@ public CursorsController(final CursorsCommitService cursorsCommitService,
         this.featureToggleService = featureToggleService;
     }
 
+    @RequestMapping(path = "/subscriptions/{subscriptionId}/cursors", method = RequestMethod.GET)
+    public ResponseEntity<?> getCursors(@PathVariable("subscriptionId") final String subscriptionId, final NativeWebRequest request) {
+        if (!featureToggleService.isFeatureEnabled(FeatureToggleService.Feature.HIGH_LEVEL_API)) {
```

Comment:
I understand your point Victor. As a result of yesterday talk we have a file which help us decide on technics we use.